lms/djangoapps/branding/__init__.py

```diff
@@ -27,6 +33,14 @@ def get_visible_courses():
 
     filtered_by_org = microsite.get_value('course_org_filter')
 
+    filtered_by_db = TileConfiguration.objects.filter(
+        enabled=True,
+    ).values('course_id').order_by('-change_date')
+
+    if filtered_by_db:
+        filtered_by_db_ids = [course['course_id'] for course in filtered_by_db]
+        filtered_by_db_keys = frozenset([SlashSeparatedCourseKey.from_string(c) for c in filtered_by_db_ids])
+        return [course for course in courses if course.id in filtered_by_db_keys]
```

Comment:
Well there shouldn't be dupes. I think the plan is to move the UI part into Sysadmin so doing the validations there would be more sensible. I'm inclined to see how we go with the Django Admin tool. We can easily fixup data in the db using Django Admin if necessary (insert, delete, edit, enable/disable).